test/integration/test_pkg_signing.py

```diff
@@ -57,6 +57,33 @@ def verify(ref, artifacts_folder, signature_folder, files, **kwargs):
     assert "VERIFYING  conan_sources.tgz" in c.out
 
 
+def test_pkg_sign_no_verify_function():
+    c = TestClient()
+    c.save({"conanfile.py": GenConanfile("pkg", "0.1").with_exports("export/*")
+            .with_exports_sources("export_sources/*").with_package_file("myfile", "mycontents!"),
+            "export/file1.txt": "file1!",
+            "export_sources/file2.txt": "file2!"})
+    signer = textwrap.dedent(r"""
+        import os
+
+        def sign(ref, artifacts_folder, signature_folder, output):
```

Comment:
I think the use case that I was considering was the opposite.
It is like that organizations distribute a package signing plugin that has only the ``verify()`` part, so their developers and most CI jobs can only verify packages. So users can do a ``conan cache check-integrity`` and verify packages (as well as verifying them during download). But they aren't allowed to sign packages (very expected to not give developers package signing permissions).